### DISCUSSION TITLE
明细表自定义列头分组如何使用隐藏明细表列头功能

### DISCUSSION THREAD

**Discussion by qianchengjie** [antvis/S2#3043] [Q&A]
当我使用自定义列头分组时，无法隐藏列头

**Comment by petercat-assistant** [comment]
目前在 antvis/S2 的问题列表中没有找到关于“自定义列头分组无法隐藏列头”的相关问题。您可以尝试以下方法来解决这个问题：

1. **检查配置**: 确保您在配置中正确设置了隐藏列头的选项。有时候，配置项可能会被忽略或设置错误。

2. **更新版本**: 确保您使用的是 antvis/S2 的最新版本。新版本可能修复了旧版本中的一些问题。

3. **查看文档**: 查阅 antvis/S2 的官方文档，看看是否有关于自定义列头分组和隐藏列头的详细说明。

4. **代码调试**: 如果您有能力，可以查看相关代码，看看是否有逻辑上的问题导致列头无法隐藏。

如果问题依然存在，建议在 antvis/S2 的 GitHub 讨论区或 issue 区提交一个新的问题，详细描述您的使用场景和遇到的问题，以便开发者能够更好地帮助您。

如需进一步帮助，请在评论中描述您的问题，并 @petercat-assistant 以开始与我对话。

**Comment by qianchengjie** [comment]
找到方法了，需要使用meta中的id字段
```js
s2.interaction.hideColumns([meta.id]);
```
